quack_app/lib/screens/settings/settings_screen.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:quack_app/constants/constants.dart';
+import 'package:quack_app/core/menu_data.dart';
+import 'package:quack_app/screens/settings/subscreens/about_screen.dart';
+import 'package:quack_app/screens/settings/subscreens/account_prefrences_screen.dart';
+import 'package:quack_app/screens/settings/subscreens/food_prefrences_screen.dart';
+import 'package:quack_app/screens/settings/subscreens/help_and_support_screen.dart';
+import 'package:quack_app/screens/settings/subscreens/notifications_prefrences_screen.dart';
+import 'package:quack_app/screens/settings/subscreens/your_calendar_screen.dart';
+
+// Settings screen implements the screen for the settings page
+class SettingsScreen extends StatefulWidget {
+  final String title;
+  const SettingsScreen({Key? key, required this.title}) : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() => _SettingsScreenState();
+}
+
+class _SettingsScreenState extends State<SettingsScreen> {
+  late MenuData menuData;
+  final TextStyle _style = const TextStyle(color: Colors.black, fontSize: 24);
+  final AutoSizeGroup _group = AutoSizeGroup();
+  @override
+  void initState() {
+    // Re-cache updated values on view
+    super.initState();
+    menuData = MenuData();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Column(
+        children: <Widget>[
+          Expanded(
+            child: Container(
+                color: Constants.kBackgroundGrey,
+                child: SingleChildScrollView(
+                  physics: const ClampingScrollPhysics(),
+                  child: Column(children: <Widget>[
+                    _getTile(
+                        context, //
+                        "Food Prefrences", //
+                        Icons.food_bank_outlined, //
+                        const FoodPrefrencesScreen()), //
+                    _getTile(
+                        context,
+                        "Notification Prefrences",
+                        Icons.notifications_active_outlined,
+                        const NotificationsPrefrencesScreen()),
+                    _getTile(
+                        context,
+                        "Your Calendar",
+                        Icons.calendar_today_outlined,
+                        const YourCalendarScreen()),
```

Comment:
Probably not an issue, but does this initialize every sub-setting view while loading this settings view?

Reply:
Potentially? But I guess the same could be said about how the Navbar works?